tests/baselines/reference/moduleAugmentationDeclarationEmit1.js

```diff
@@ -0,0 +1,68 @@
+//// [tests/cases/compiler/moduleAugmentationDeclarationEmit1.ts] ////
+
+//// [map.ts]
+
+import { Observable } from "./observable"
+
+(<any>Observable.prototype).map = function() { }
```

Comment:
I would help if conversations like these bore their fruits as extensive comments in the code.

If the members of the core team interpret that stuff differently, imagine the confusion of an external observer. Consider a day when both of you have left the team and no one knows `<any>` reason to have that cast in there.